setup/StackSetup/azurecmdfiles.wxi

```diff
@@ -0,0 +1,1207 @@
+                        </Component>
+                        <Component Id="cmpA7033E45EA31DB2536CA37A4A3C8FF79" Guid="*">
+                            <File Id="fil90C35CA078B4987AE75188F53FAB7A8E" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.AzureStackStorage\Microsoft.WindowsAzure.Commands.Common.dll" />
+                        </Component>
+                        <Component Id="cmp13B432C58CCE93F833BF291FEBE890FE" Guid="*">
+                            <File Id="fil6199B2627508C998509D104B0D4808B3" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.AzureStackStorage\Microsoft.WindowsAzure.Management.dll" />
+                        </Component>
+                        <Component Id="cmp27B0788FBA03ECEBDB4A1A6FEE425DBE" Guid="*">
+                            <File Id="filB7C76763513F8A4625090D780A4234DC" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.AzureStackStorage\Microsoft.WindowsAzure.Storage.dll" />
+                        </Component>
+                        <Component Id="cmpC1B4A054FDF189529C7F141DFF34F627" Guid="*">
+                            <File Id="fil5CE63C059CEB87D033F4CB5F72E368D8" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.AzureStackStorage\Newtonsoft.Json.dll" />
+                        </Component>
+                        <Component Id="cmp65415E5150B588AE9E3C610865988601" Guid="*">
+                            <File Id="fil0233B880D82CAB2CE92A78465E7A3517" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.AzureStackStorage\ResourceManagerStartup.ps1" />
+                        </Component>
+                        <Component Id="cmp202324C8EAC333C726E21AAA9113D93C" Guid="*">
+                            <File Id="filB509D5A80ED92B51EC6BDFDED39326DC" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.AzureStackStorage\System.Net.Http.Extensions.dll" />
+                        </Component>
+                        <Component Id="cmp78651F768E6FEC29FB7B314844A2011D" Guid="*">
+                            <File Id="fil6F69DB15BB3D65A36AB4E06388796654" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.AzureStackStorage\System.Net.Http.Primitives.dll" />
+                        </Component>
+                        <Component Id="cmpF1D2745AF8971AC4B7E570D509984890" Guid="*">
+                            <File Id="filBD3E5ED799CF21908B9AAB3D356C7F1C" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.AzureStackStorage\System.Spatial.dll" />
+                        </Component>
+                    </Directory>
+                    <Directory Id="dir470D08462380F56397F9D0288EEA2E52" Name="AzureRM.Compute">
```

Comment:
So, don't we just want the AzureSTack packages as part of thsi msi, and not the client libraries?